vagrant/provisioning/roles/arkcase-app/templates/arkcase-commons-server.yaml

```diff
@@ -1,18 +1,24 @@
 arkcase:
   baseURL: "https://{{ external_host }}/arkcase"
   basePortalURL: "https://{{ external_host }}/portal"
-  fileManagementURL: "https://{{ external_host }}"
+  fileManagementURL: "https://{{ internal_host }}:{{ file_management_port | default(9068) }}"
+  fileUtilsURL: "https://{{ internal_host }}:{{ file_utils_port | default(9078) }}"
```

Comment:
was this called fileUtilsMicroserviceURL before?  What value is the arkcase-file-utils service looking for?

Reply:
Yes. I updated the field name and updated in the prc extension.